### PR TITLE
cappuccino and passione now do things

### PR DESCRIPTION
**Pull request by SonixApache**
🆑 
 - tweak: cappuccino now heals like milk does & passione now heals like low grade honey does

### CODE REVIEW

**Comment by boy2mantwicethefam** [comment]
is that a jojo reference

**Comment by Kurfursten** [comment]
I'm going to say this is an oversight. It makes sense that honey/milk shouldn't "lose" their properties by being mixed into coffee. At most it would be diluted/lessened.

I guess on the other hand this means it now stacks with honey so you can do passione+honey for double heals, though. Maybe we need a var that makes one reagent prevent the other from processing.

**Comment by Zth--** [comment]
We should have a proc that handles nutrition instead of doing `nutrition +=` directly

File: code/modules/reagents/Chemistry-Reagents.dm [hunk 1]
```diff
@@ -6628,6 +6633,23 @@ var/global/list/tonio_doesnt_remove=list("tonio", "blood")
...
 	id = PASSIONE
 	description = "Rejuvenating!"
+	nutriment_factor = 3 * REAGENTS_METABOLISM //because honey
+	
+/datum/reagent/drink/coffee/passione/on_mob_life(var/mob/living/M)
+	..()
+
+	if(ishuman(M))
+		var/mob/living/carbon/human/H = M
+		if(!holder)
```
Comment (Probe1) [inline code/modules/reagents/Chemistry-Reagents.dm:6643]: Its strange that this kind of sanity isn't like, way the fuck higher up.

**Pull request merged**